doc/releases/changelog-dev.md

```diff
@@ -17,6 +17,9 @@
 
 <h3>Improvements 🛠</h3>
 
+* Added decomposition to ``qml.QubitUnitary`` for any number of qubits.
```

Comment:
```suggestion
* A decomposition rule has been added to ``qml.QubitUnitary`` that works for any number of qubits.
```

Can we show what the decomposition rule is, maybe? You can print the decomposition rule with `print(qml.list_decomps(qml.QubitUnitary)[index])` and it will show the quantum function + resources registered :) 

Is there a benefit to using this decomposition outside of it being general to the number of qubits?

Reply:
It's cheaper than using the default select decomposition in the case where we have multiple Multiplexers in the circuit because we can prepare a single resource state and re-use it for each multiplexer.

Reply:
```suggestion
* A more resource efficient decomposition rule has been added to :class:`~.QubitUnitary` that works for 
  any number of qubits.
```